From c2d279941ae256c12751ca9157a4aafbb45c72f7 Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Sun, 30 Apr 2017 11:22:18 +0800
Subject: [PATCH 61/69] arm64: allwinner: a64: add simplefb node for LCD

Pinebook uses the LCD controller of A64 SoC.

Add simplefb node for it.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 arch/arm64/boot/dts/allwinner/sun50i-a64.dtsi | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-a64.dtsi
index ec440e84a714..ee9443d3c5c3 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64.dtsi
@@ -65,6 +65,15 @@
 				 <&ccu CLK_TCON1>, <&ccu CLK_HDMI>;
 			status = "disabled";
 		};
+
+		simplefb_lcd: framebuffer@1 {
+			compatible = "allwinner,simple-framebuffer",
+				     "simple-framebuffer";
+			allwinner,pipeline = "de0-lcd0";
+			clocks = <&ccu CLK_BUS_TCON0>, <&ccu CLK_BUS_DE>,
+				 <&ccu CLK_DE>, <&ccu CLK_TCON0>;
+			status = "disabled";
+		};
 	};
 
 	cpus {
-- 
2.12.2

